### ISSUE TITLE
Could not find a suitable TLS CA certificate bundle, invalid path

### ISSUE THREAD

**Issue by iamdank** [seik/stilio#3]
Hi there

Newbie programmer here. On Ubuntu 20.04, having difficulty compiling stilio/dht. Upon command > sudo docker-compose up:

"OSError

  Could not find a suitable TLS CA certificate bundle, invalid path: /usr/local/lib/python3.8/site-packages/certifi/cacert.pem

  at /usr/local/lib/python3.8/site-packages/requests/adapters.py:227 in cert_verify

ERROR: Service 'crawler' failed to build: The command '/bin/sh -c poetry install --no-interaction --no-dev' returned a non-zero code: 1"

I have manually copied cacert.pem to  "/usr/local/lib/python3.8/site-packages/certifi/cacert.pem"
It didn't work.

Is there something I have missed?

**Comment by seik** [comment]
Hi @iamdank 

I saw this problem but have no clue how to fix it at the moment. A temporary fix is to remove `sphinx` and `sphinx-rtd-theme` from `pyproject.toml`.

In case someone tries to fix this, this error is related to `requests` depending on `certifi`. Things I tried to fix this without success:

- Update the docker image to depend on `python:3.8-slim-buster`, `python:3.8-slim`, `python:3.8-buster`
- Perform a `update-ca-certificates`
- https://stackoverflow.com/questions/41328451/ssl-module-in-python-is-not-available-when-installing-package-with-pip3/44758621
